### ISSUE TITLE
netbird client The windows 7 is not supported ?

### ISSUE THREAD

**Issue by szzylph** [netbirdio/netbird#1735]
netbird client The windows 7 is not supported ?

    windows 7 After the installation of netbrid, the start flash is gone, the service is not started, the tray icon is not there.

**Comment by szzylph** [comment]
C:\Windows\system32>netbird service start
Exception 0xc0000005 0x8 0x0 0x0
PC=0x0

runtime.asmstdcall()
        /opt/hostedtoolcache/go/1.21.8/x64/src/runtime/sys_windows_amd64.s:65 +0x75 fp=0x22fca0 sp=0
x22fc80 pc=0x46e375
rax     0x0
rbx     0x1c44da0
rcx     0x21c43a0
rdi     0x7fffffde000
rsi     0x22fea0
rbp     0x22fde0
rsp     0x22fc78
r8      0x0
r9      0x22fee0
r10     0x1c94db8
r11     0x21
r12     0x22fec0
r13     0x1
r14     0x1c44340
r15     0x0
rip     0x0
rflags  0x10293
cs      0x33
fs      0x53
gs      0x2b

**Comment by lixmal** [comment]
Yes, we are using go1.21 which doesn't support Windows7. There's currently no build provided that does. You could try compiling the client with go1.20, but no guarantee.

**Comment by i486** [comment]
I'm not a developer, but from what I can gather from this [discussion](https://rb.gy/sfjbpi), it seems like there's only one API change that broke Go on Windows 7. After reading the entire conversation, it looks like many of the main Go developers are open to make Go compatible with Windows 7 again. They're just waiting for a volunteer to handle it through proper channels.

**Comment by stunndard** [comment]
Try this: https://github.com/stunndard/golangwin7patch

**Comment by lw3eov** [comment]
> Try this: https://github.com/stunndard/golangwin7patch

The "[download it from Releases](https://github.com/stunndard/golangwin7fixer/releases)" link is broken.

**Comment by stunndard** [comment]
thanks, fixed!
